indigo/management/commands/datamigrationcleancountry.py

```diff
@@ -0,0 +1,212 @@
+import jsondataferret
+from django.core.management.base import BaseCommand
+from jsondataferret.pythonapi.newevent import NewEventData, newEvent
+
+from indigo.models import Project
+
+
+class Command(BaseCommand):
+    help = "Data Migration - Clean Country"
```

Comment:
I'm not sure if this really explains the command. Perhaps something about converting to country codes?

Reply:
Point, but as this should be run once then deleted I'll just go with it